pallets/parachain-staking/src/weights.rs

```diff
@@ -217,9 +215,9 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking Total (r:1 w:1)
 	#[rustfmt::skip]
 	fn execute_leave_candidates(x: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 87_000
-			.saturating_add(Weight::from_ref_time(31_860_000 as u64).saturating_mul(x as u64))
+		Weight::from_ref_time(120_885_000 as u64)
```

Comment:
That's a significant increase

Reply:
There is a change here since the last benhcmarks: https://github.com/PureStake/moonbeam/pull/1828